privacyidea/lib/tokens/certificatetoken.py

```diff
@@ -168,6 +325,27 @@ def get_class_info(key=None, ret='all'):
                            'type': 'int',
                            'desc': _("The user may only have this maximum number of active certificates assigned."),
                            'group': GROUP.TOKEN
+                       },
+                       ACTION.REQUIRE_ATTESTATION: {
+                           'type': 'string',
+                           'desc': _("Enrolling a certificate token can require an attestation certificate. "
+                                     "(Default: ignore)"),
+                           'group': GROUP.TOKEN,
+                           'value': [""]
```

Comment:
Please add the `REQUIRE_ACTIONS` here

Reply:
This seems to have vanished.

Will do

Reply:
Done in b94a7c2cf281652d3be7957a85ebb9549b4ad3b3